From 7019dcb30bfdf9929332e73a928aee2b854348ba Mon Sep 17 00:00:00 2001
From: Mark Harfouche <mark.harfouche@gmail.com>
Date: Thu, 19 Dec 2024 09:28:00 -0500
Subject: [PATCH] use __conda_activate reactivate instead of __conda_reactivate

The work in https://github.com/mamba-org/mamba/pull/3643 was deleted

this is really hurting users at conda-forge (myself included)
https://github.com/conda-forge/miniforge/issues/700#issuecomment-2554272743
---
 mamba/mamba/shell_templates/mamba.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mamba/mamba/shell_templates/mamba.sh b/mamba/mamba/shell_templates/mamba.sh
index 6b1c1640a7..a3659358d5 100644
--- a/mamba/mamba/shell_templates/mamba.sh
+++ b/mamba/mamba/shell_templates/mamba.sh
@@ -17,7 +17,7 @@ mamba() {
             ;;
         install|update|upgrade|remove|uninstall)
             __mamba_exe "$@" || \return
-            __conda_reactivate
+            __conda_activate reactivate
             ;;
         *)
             __mamba_exe "$@"
